dbms/src/Operators/Operator.h

```diff
@@ -87,10 +84,26 @@ class Operator
         header = header_;
     }
 
+    const OperatorProfileInfoPtr & getProfileInfo() const { return profile_info_ptr; }
+
+    virtual IOProfileInfoPtr getIOProfileInfo() const { return IOProfileInfo::createForLocal(profile_info_ptr); }
```

Comment:
Why not throw error for the default impl?

Reply:
ok.
Now the default implementation throws an error, and add `IOProfileInfoPtr getIOProfileInfo() const override { return IOProfileInfo::createForLocal(profile_info_ptr); }` for `AddExtraTableIDColumnTransformOp`, `DMSegmentThreadSourceOp`, `NullSourceOp` and `UnorderedSourceOp`